arm/Microsoft.Storage/storageAccounts/deploy.bicep

```
@@ -138,6 +138,18 @@ param diagnosticMetricsToEnable array = [
   'Transaction'
 ]
 
+@description('Optional. The resource ID of a key vault to reference a customer managed key for encryption from.')
+param cMKKeyVaultResourceId string = ''
+
+@description('Optional. The name of the customer managed key to use for encryption. Cannot be deployed together with the parameter \'systemAssignedIdentity\' enabled.')
+param cMKeyName string = ''
```

Same comment as before.
```suggestion
param cMKKeyName string = ''
```

---

done